pkg/chains/formats/slsa/v1/internal/material/material_test.go

```diff
@@ -118,13 +117,41 @@ func TestRemoveDuplicates(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "same uri but different digest, swap order",
+		mats: []slsa.ProvenanceMaterial{
+			{
+				URI: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init",
+				Digest: slsa.DigestSet{
+					"sha256": "b963f6e7a69617db57b685893256f978436277094c21d43b153994acd8a01248",
+				},
+			}, {
+				URI: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init",
+				Digest: slsa.DigestSet{
+					"sha256": "b963f6e7a69617db57b685893256f978436277094c21d43b153994acd8a01247",
+				},
+			},
+		},
+		want: []slsa.ProvenanceMaterial{
+			{
+				URI: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init",
+				Digest: slsa.DigestSet{
+					"sha256": "b963f6e7a69617db57b685893256f978436277094c21d43b153994acd8a01248",
+				},
+			}, {
+				URI: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init",
+				Digest: slsa.DigestSet{
+					"sha256": "b963f6e7a69617db57b685893256f978436277094c21d43b153994acd8a01247",
+				},
+			},
+		},
 	}}
 	for _, tc := range tests {
 		mat, err := RemoveDuplicateMaterials(tc.mats)
 		if err != nil {
 			t.Fatalf("Did not expect an error but got %v", err)
 		}
-		if diff := cmp.Diff(tc.want, mat, test.OptSortMaterial); diff != "" {
```

Comment:
if `test.OptSortMaterial` is no longer needed now then can we remove it from the test folder and the e2e tests as well (since we are cleaning it up) ?

Reply:
ie. https://github.com/tektoncd/chains/blob/main/test/sortoptions.go
and https://github.com/tektoncd/chains/blob/9950cb5dbc93493daab02af348754fc0515a11c1/test/examples_test.go#L139

Reply:
Good call! Done.